test/minioptest_failures_dict.json

```diff
@@ -0,0 +1,44 @@
+{
+  "_description": "This is a dict containing failures for tests autogenerated by generate_opcheck_tests. For more details, please see https://docs.google.com/document/d/1Pj5HRZvdOq3xpFpbEjUZp2hBovhy7Wnxw14m6lF2154/edit",
+  "_version": 1,
+  "data": {
+    "aten::mm": {},
+    "aten::nonzero": {
+      "test_aot_dispatch_static__test_nonzero": {
+        "comment": "",
+        "status": "xfail"
+      }
+    },
+    "aten::sin_": {},
+    "mini_op_test::delayed_error": {
+      "test_aot_dispatch_dynamic__test_delayed_error": {
+        "comment": "",
+        "status": "xfail"
+      },
+      "test_aot_dispatch_static__test_delayed_error": {
+        "comment": "",
+        "status": "xfail"
+      }
+    },
+    "mini_op_test::incorrect_schema": {
+      "test_schema__test_incorrect_schema": {
+        "comment": "",
+        "status": "xfail"
+      }
+    },
+    "mini_op_test::no_abstract": {
+      "test_aot_dispatch_dynamic__test_no_abstract": {
+        "comment": "",
+        "status": "xfail"
+      },
+      "test_aot_dispatch_static__test_no_abstract": {
+        "comment": "testing a skip just for the fun of it",
+        "status": "skip"
+      },
+      "test_faketensor__test_no_abstract": {
+        "comment": "",
+        "status": "xfail"
+      }
+    }
+  }
+}
```

Comment:
would be nice to do the trailing newline correctly

Reply:
yup